Update full tutorials from develop (#567)
Avaiga/taipy-doc: docs/tutorials/complete_applications/complete_application/index.md

!!! important "Supported Python versions"

    Taipy requires **Python 3.8** or newer.

> Welcome to the **Tutorial** guide to create a complete application using Taipy frontend and backend. No prior knowledge is required to complete this Tutorial.

Reviewer: ```suggestion
Welcome to the **Tutorial** guide to create a complete application from frontend to
backend. No prior knowledge is required to complete this Tutorial.
```

> Welcome to the **Tutorial** guide to create a complete application using Taipy frontend and backend. No prior knowledge is required to complete this Tutorial.

Reviewer 2: ```suggestion
Welcome to the **Tutorial** guide on how to make a full application using Taipy for both the front and back ends. You don't need any previous knowledge to finish this tutorial.
```

![Tutorial application](step_01/overview.gif){ width=700 style="margin:auto;display:block;border: 4px solid rgb(210,210,210);border-radius:7px" }

> Each step of the **"Tutorial"** focus on basic concepts of *Taipy*. One step aims to create and complete only one file.

Reviewer: ```suggestion
```

> Each step of the **"Tutorial"** focus on basic concepts of *Taipy*. One step aims to create and complete only one file.

Reviewer 2: ```suggestion
In the **"Tutorial"** each step concentrates on fundamental ideas about *Taipy*. Each step is designed to create and finish just one file.
```

    [Python installation guide](http://docs.python-guide.org/en/latest/starting/installation/)
    can guide you through the process.

Once Taipy is installed, you can use the CLI to create the scaffolding. The default template will be used throughout the creation of the application.

``` console
> taipy create --template default
Application root folder name [taipy_application]:
Application main Python file [main.py]:
Application title [Taipy Application]:
Page names in multi-page application? []: data_viz scenario performance
Does the application use scenario management or version management? [No]: yes
Does the application use Rest API? [No]: no
```

So, without further delay, let's begin to code!

## Steps

> 1. [Data Visualization page](step_01/ReadMe.md)

Reviewer: Please rename each file step_XX/step_XX.md or at least step_XX/index.md

2. [Algorithms used](step_02/ReadMe.md)

> 3. [Backend Configuration](step_03/ReadMe.md)

Reviewer: ```suggestion
3. [Scenario Configuration](step_03/ReadMe.md)
```
